eta/modules/sample_videos.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 '''
-Module for (re)sampling videos.
+A module for (re)sampling videos.
```

Comment:
This should be more thorough and clarify whether or not spatial resampling can be done (yes, via out_opts, I believe) in addition to temporal.

Reply:
`resize_videos` is for spatial and `sample_videos` is for temporal, although anything could be done in principle with `ffmpeg_out_opts`, yes.

Alas, these modules should probably be merged into one spatio-temporal module for efficiency... but I don't feel like doing it now.

Reply:
The first line of ETA module docstrings (the "short description" in Google docstring lingo) is shoved into the description field of the metadata file, so I have been keeping them short. Although we should brainstorm what exactly we want in these descriptions and what we'll do with them.

For the benefit of developers, I should also include a long description that discusses the module in more detail.